controllers/provider-azure/charts/internal/cloud-provider-config/templates/cloud-provider-config-acr.yaml

```diff
@@ -0,0 +1,17 @@
+{{if hasKey .Values "acrIdentityClientId" -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cloud-provider-acr-config
```

Comment:
Should we rather call it `kubelet-acr-config`? Or will somebody else use this as well?